### ISSUE TITLE
Please help with input.harbor

### ISSUE THREAD

**Issue by inghoward** [savonet/liquidsoap#537]
Hi friends,
I'm a newbie using liquidsoap, so there are some thing that I'm not able to make it work.
My system:
VPS with ubuntu 14.04. Liquidsoap 1.3.3 installed via OPAM.

Liquidsoap is working using a playlist and streaming to SHOUTcast, but now I want to make a live show from my pc using mixxx.

My script
**********************************************************************
#!/home/howard/.opam/system/bin/liquidsoap

set("log.file.path","/tmp/brandom.log")

set("init.daemon.pidfile.path", "/tmp")

set("log.stdout",true)

lista = 
  smart_crossfade(
      normalize(
        playlist(reload=43200, mode="random", "/home/howard/playlist.m3u") 
        )
  )

#live = input.harbor("/",port=64125,password="hackme",icy=true,on_connect=on_harbor_connect)

output.shoutcast(
  %mp3(bitrate=128),
  id="1",
  host="xxx.xxx.xxx.xxx",
  port = 80,
  password = "mypassword",
  user = "source",
  genre = "Rock",
  url = "http://myurl.com",
  name = "My Radio Name",
  public = true,
  irc = "myirc_irc",
  mksafe(
    lista
  )
)
**********************************************************
Everything works until I uncomment
#live = input.harbor("/",port=64125,password="hackme",icy=true,on_connect=on_harbor_connect)

I've noticed that when I uncomment the line above no log is generated. 

May be I need to install an additional module to make it work.

Any help?

**Comment by svandragt** [comment]
I think you'll to at least have remove `,on_connect=on_harbor_connect` from the input.harbor line as you haven't defined an `on_harbor_connect` function

**Comment by inghoward** [comment]
Thanks for your response @svandragt . Unfortunately still not resolve my issue. I've deleted the on_connect=on_harbor_connect parameter. Now the script run, but I can't connect using mixxx. I've tested to see if port 64125 is opened, and I noticed that is not. 

If I use this line "set("harbor.bind_addr","0.0.0.0")" got this warning:
[lang.deprecated:2] WARNING: "harbor.bind_addr" is deprecated! Please use "harbor.bind_addrs" 

If I use "set("harbor.bind_addrs","0.0.0.0")" then script never runs.

**Comment by jonathanstowe** [comment]
Shouldn't it be ```set("harbor.bind_addrs",["0.0.0.0"])``` as it is a list?

**Comment by toots** [comment]
Yeah, @jonathanstowe is correct. Also, this is the default value so your issue probably doesn't come from this.

Do you have any log to share with us?
